osgeo_importer/forms.py

```diff
@@ -1,9 +1,81 @@
+import os
 from django import forms
 from .models import UploadFile
+import zipfile
+import tempfile
 
 
-class UploadFileForm(forms.ModelForm):
+class XXXUploadFileForm(forms.ModelForm):
```

Comment:
Rather just delete things like this than rename it. It's still in the git history after all